static/src/js/components/ProjectCollections/ProjectCollectionsItem.js

```diff
@@ -107,16 +114,22 @@ const ProjectCollectionItem = ({
         </MoreActionsDropdown>
       </div>
       <ul className="project-collections-item__stats-list">
-        <li
-          className="project-collections-item__stats-item project-collections-item__stats-item--granule-count"
-        >
-          {`${abbreviate(granuleCount, 1)} Granules`}
-        </li>
-        <li
-          className="project-collections-item__stats-item project-collections-item__stats-item--total-size"
-        >
-          {`Est. Size ${size} ${unit}`}
-        </li>
+        {
+          (typeof granuleCount !== 'undefined' && size && unit) && (
```

Comment:
I'm not necessarily asking for a change here, but I think we need a util for this kind of thing, or maybe lodash has something? checking `typeof` seems undesirable.

Reply:
does `''` evaluate? I see size and unit default to empty strings above. Do we have an `isLoading` for this to check on something instead of checking these three variables? It seems fragile 🤷‍♂

Reply:
`''` will evaluate to false. I can look into a replacement for the `undefined` check. We dont currently have loading states in place, but we have a ticket coming up to implement loading states. I plan to do exactly that in that ticket.